lib/mix/lib/mix/tasks/deps.compile.ex

```diff
@@ -355,7 +374,7 @@ defmodule Mix.Tasks.Deps.Compile do
   defp shell_cmd!(%Mix.Dep{app: app} = dep, config, command, env \\ []) do
     if Mix.shell().cmd(command, [print_app: true] ++ opts_for_cmd(dep, config, env)) != 0 do
       Mix.raise(
-        "Could not compile dependency #{inspect(app)}, \"#{command}\" command failed. " <>
+        "Could not compile dependency #{inspect(app)}, \"#{inspect(command)}\" command failed. " <>
```

Comment:
Unrelated change?

In any case, `inspect` already adds double quotes, so we should either have `inspect` or the double quotes. :)

Reply:
This raised an error when "stringifying" the command (no implementation for it). So I'll remove the quotes.